src/Config.js

```diff
@@ -59,16 +51,16 @@ class Config {
+  static get awsCognitoUserPoolId() {
+    return process.env.COGNITO_USER_POOL_ID || 'us-west-2_CGd9Wq136'
   }
 
-  static get awsCognitoIdToken() {
-    return 'eyJraWQiOiI1OFRSSmJ3bFc4RlJJTmFiZllxak11d0V2empcLzBiMjBtTXBNejF2S1FUaz0iLCJhbGciOiJSUzI1NiJ9.eyJhdF9oYXNoIjoiUEJDLTdsazlITWtwYkpjdjlXYlluZyIsInN1YiI6Ijc4OWRkYTdkLTI1YzAtNGE4Zi05YzYyLWIzMTE2YTk3Y2M5YiIsImF1ZCI6IjJ1NnM3cHFrYzFncnExcXM0NjRmc2k4MmF0IiwiZW1haWxfdmVyaWZpZWQiOnRydWUsImV2ZW50X2lkIjoiNjNjMjJhYmYtNWZkNi0xMWU5LWEwN2ItZTlmZTA3NzQxN2ZkIiwidG9rZW5fdXNlIjoiaWQiLCJhdXRoX3RpbWUiOjE1NTUzNzEwMDEsImlzcyI6Imh0dHBzOlwvXC9jb2duaXRvLWlkcC51cy13ZXN0LTIuYW1hem9uYXdzLmNvbVwvdXMtd2VzdC0yX0NHZDlXcTEzNiIsImNvZ25pdG86dXNlcm5hbWUiOiJzaW5vcGlhLWRldnNfY2xpZW50LXRlc3RlciIsImV4cCI6MTU1NTM3NDYwMSwiaWF0IjoxNTU1MzcxMDAxLCJlbWFpbCI6InNpbm9waWEtZGV2cytjbGllbnQtdGVzdGVyQGxpc3RzLnN0YW5mb3JkLmVkdSJ9.JMf4p2teIsL3DDFgHsaqdAEd_kf0WytEsi00TsfhAIzVUglAvAYg5OZkFToE1vEjzvZH9HXhNSj6PT2DpO5HPhuWLvbGyw7dX_MfZpxO7wlIBhgy2FhRLkcytgEzyfMQEtDvRMD0-HUVpGBjwV4bteILl0cwi_MZ8y0U81Etn7QTDSUPE1bilGOBFKveLw8lzgiNJcJkRZbRc2FVoPvfVqSoq8iH2FXuw1HoY7lBWIBVAeq8ETBxBRHJc7xoN7S8YcARXnsyFJrmdAgHFs-rt5639ts1eNlXnen6sBIlVhOwaekBZgX0twEg8vZvsNK0L3GRjyVDNPB6f281DDfQtg.QGFiYyQxMjM'
+  static get cognitoTestUserName() {
+    return process.env.COGNITO_TEST_USER_NAME
   }
 
-  static get awsCognitoAccessToken() {
-    return 'eyJraWTigJ064oCdQUJDMTIzPSIsImFsZyI6IlJTMjU2In0.eyJhdF9oYXNo4oCdOuKAnWExYjJjM+KAnSzigJ1zdWLigJ064oCdYTEtYjItYzMtZDQtZTXigJ0s4oCdYXVk4oCdOuKAnWExYjJjM2Q0ZTXigJ0s4oCdZW1haWxfdmVyaWZpZWQiOnRydWUsImV2ZW50X2lk4oCdOuKAnWExLWIyLWMzLWQ04oCdLOKAnXRva2VuX3VzZSI6ImlkIiwiYXV0aF90aW1l4oCdOjU1NTU1NTU1NTUs4oCdaXNzIjoiaHR0cHM6XC9cL2NvZ25pdG8taWRwLnVzLXdlc3QtMi5hbWF6b25hd3MuY29tXC91cy13ZXN0LTJfQ0dkOVdxMTM2IiwiY29nbml0bzp1c2VybmFtZSI6InNpbm9waWEtZGV2c19jbGllbnQtdGVzdGVyIiwiZXhw4oCdOjU1NTU1NTU1NTUs4oCdaWF04oCdOjU1NTU1NTU1NTUs4oCdZW1haWwiOiJzaW5vcGlhLWRldnMrY2xpZW50LXRlc3RlckBsaXN0cy5zdGFuZm9yZC5lZHUifQ.QGFiYyQxMjM'
+  static get cognitoTestUserPass() {
+    return process.env.COGNITO_TEST_USER_PASS
```

Comment:
credentials available on request (and in circle-ci, though not in a human readable way).  didn't want to put them in a public repo, even if it is just a test user.

Reply:
put in shared_configs???